mpm/crawler.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import pathlib 
+from pathlib import Path 
+
+baseURL = 'https://www.mpm.mp.br/sistemas/consultaFolha/php/'
+
+beneficiary_types = {1: 'Membros ativos',
+                     2: 'Membros inativos',
+                     3: 'Servidores ativos',
+                     4: 'Servidores inativos',
+                     5: 'Pensionistas',
+                     6: 'Colaboradores'}
+
+# Generate endpoints able to download
+def links(month, year):
```

Comment:
Melhor retornar um dicionário de beneficiary_type -> link. Isso vai tornar o for abaixo bem mais simples

Reply:
Você viu esse comentário? O que achou?